src/components/ActionsPanel/ActionPanel.tsx

```diff
@@ -68,14 +74,21 @@ export const ActionPanel = () => {
           <DetailsHeading>
             <Icon width={20} type={currentAction.icon} />
             <HeadingText>{currentAction.heading}</HeadingText>
+            <Icon width={20} type="closed-eye" onClick={dismiss} />
```

Comment:
Can we have it use the clickable pointer? So it is obvious that it can be clicked.